src/formatter/date.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  DateFromString,
+  DateFromTimestamp,
+  UtcOnlyIsoDateFromString
+} from "@pagopa/ts-commons/lib/dates";
+import * as E from "fp-ts/Either";
+import { identity, pipe } from "fp-ts/function";
+import * as t from "io-ts";
+import { failure } from "io-ts/PathReporter";
+
+const onLeft = (errors: t.Errors): string =>
+  pipe(failure(errors), e => e.join("\n"));
+
+export const dateStringToUtcFormat = (dateString: string): string =>
+  pipe(
+    dateString,
+    DateFromString.decode,
+    E.match(onLeft, date => date.toUTCString())
```

Comment:
I think that maybe is better to return an Either<Error, string>  to handle a possible decode error and treat it as an error instead of a string.
The same for the other methods. What do you think @AleDore ?

Reply:
agree